### DISCUSSION TITLE
shared storage.conf location for rootless podman

### DISCUSSION THREAD

**Discussion by hmoffatt** [containers/podman#19807] [Q&A]
I'd like to configure storage.conf for all users to use with rootless podman.  Is there a default storage.conf location that is used if `$HOME/.config/containers/storage.conf` doesn't exist?

I tried to modify `/etc/containers/storage.conf` but changes there (specifically `additionalimagestores` ) don't seem to be used when running rootless.

`/usr/share/containers/storage.conf` is read I think, but that belongs to the package system and changes would be lost on package upgrade.

**Comment by edsantiago** [comment]
>changes there (specifically additionalimagestores ) don't seem to be used

I independently noticed this yesterday. It works if you do `env CONTAINERS_STORAGE_CONF=/etc/containers/storage.conf`. I haven't taken the time to track down why, but if you have the inclination, I would encourage you to file that as a bug.

**Reply by hmoffatt** [reply]
I figured it wasn't reading `/etc/containers/storage.conf` at all, but strace shows it is.

**Reply by hmoffatt** [reply]
I've opened #19827
